src/javascripts/services/accountService.js

```diff
@@ -30,6 +33,10 @@ class AccountService{
     */
     isValidAccount = ({email, password}) => {
         const validAccount = this.accountList.some((item) => item.email === email && item.password === password)
```

Comment:
For boolean, should use prefix is/has/can/...

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!